modules/representation_management/app/controllers/representation_management/v0/pdf_generator_2122_controller.rb

```diff
@@ -25,14 +25,12 @@ def create
       private
 
       def form_params
-        params.require(:pdf_generator2122).permit(
-          params_permitted.unshift(:organization_name)
-        )
+        params.require(:pdf_generator2122).permit(params_permitted)
       end
 
       def flatten_form_params
         {
-          organization_name: form_params[:organization_name],
+          organization_name: form_params[:representative][:organization_name],
```

Comment:
I don't think it's a good idea to have the front end send us something labeled as an organization name when the content is an id and it is used as an id later on.

Reply:
I've converted the incoming parameter to `organization_id` and @cosu419 confirmed that he could make the front end request match that.

The data object will still have an `organization_name` method that will lookup the name from the database.